routes/index.js

```diff
@@ -9,19 +9,21 @@ import hierarchyRoute from "./Hierarchy/index.js";
 import roleRoute from "./role/index.js"
 import jobRoute from "./job/index.js";
 import permissionRoute from "./permission/index.js";
-
 import employeeRoute from "./employee/index.js";
+import performanceRoute from "./performance/index.js"
 
 const protectedRouter = express.Router();
 const unProtectedRouter = express.Router();
 
 // Protected Routes
+protectedRouter.use("/performance", performanceRoute);
+protectedRouter.use("/misc", miscRoute);
+
 
 // Un-Protected Routes
 unProtectedRouter.use("/user", userRoute);
 unProtectedRouter.use("/employee", employeeRoute);
```

Comment:
It seems that your are not following today's msgs. @Azmat7860

Reply:
Make all routes private except signIn/signUp.

Reply:
Sorry Sir 
But now it is done.
Please review it now